operations/helm/charts/grafana-agent/templates/containers/_agent.yaml

```diff
@@ -2,13 +2,19 @@
 - name: grafana-agent
   image: {{ .Values.image.repository }}:{{ include "grafana-agent.imageTag" . }}
   args:
+    {{- if eq .Values.agent.mode "flow"}}
     - run
     - /etc/agent/{{ include "grafana-agent.config-map.key" . }}
     - --storage.path={{ .Values.agent.storagePath }}
     - --server.http.listen-addr={{ .Values.agent.listenAddr }}:{{ .Values.agent.listenPort }}
     {{- if not .Values.agent.enableReporting }}
     - --disable-reporting
     {{- end}}
+    {{- end}}
+    {{- if eq .Values.agent.mode "static"}}
+    - -config-file=/etc/agent/{{ include "grafana-agent.config-map.key" . }}
```

Comment:
Think this should be `config.file` instead of `config-file` :)

Reply:
Naturally. 🙈 thank you! Adapted it.

Reply:
it's work, what about the reload config cmd ? 
https://github.com/grafana/agent/blob/8c953cfd8ffb8aba4cbee2638cc077d3ba0ff6f8/operations/helm/charts/grafana-agent/templates/containers/_watch.yaml#L11